### PR TITLE
Monitoring single group

### PR DESCRIPTION
**Pull request by eozan**

### CODE REVIEW

**Comment by amagdas** [comment]
LGTM :+1:

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk-monitoring/monitoring.js [hunk 1]
```diff
@@ -139,12 +147,14 @@
                 scope.fetching = false;
                 scope.cacheNextItems = [];
                 scope.cachePreviousItems = [];
+                scope.limited = monitoring.singleGroup ? false : true;
```
Comment (plamut) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: You can just say `!monitoring.singleGroup` :wink:

**Comment by plamut** [comment]
Guys, we forgot about the test. :wink:

**Pull request merged**